src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -896,6 +940,8 @@
     <NugetPack Nuspecs="$(NuSpecPath)"
                OutputDirectory="$(PackageOutputPath)"
                ExcludeEmptyDirectories="true"
+               PackSymbolPackage="true"
+               SymbolPackageOutputDirectory="$(SymbolPackageOutputPath)"
                Condition="'$(_SkipCreatePackage)' != 'true'"/>
     <!-- Create a marker that records the path to the generated package -->
     <WriteLinesToFile Lines="$(PackageOutputPath)$(Id).$(Version).nupkg"
```

Comment:
Ah, I'll add that. Does it matter if the line has a file that doesn't exist? (Wondering if I need to handle the case where `NugetPack` decides not to create a package because there are no symbols or sources.)